### PR TITLE
MTM-59331 add changelog for databroker robustness fixes

### PR DESCRIPTION
**Pull request by tjegorow-c8y**
Changelog for https://github.softwareag.com/IOTA/cumulocity-core/pull/2861

### CODE REVIEW

**Review by schmiel-cumulocity — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-501-fixes-to-increase-databroker-robustness.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-V6J_FcOT2
+    label: Data broker
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-59331
+version: 1020.501.0
+---
+There was an issue where data broker connectors stopped forwarding messages due to the platform network reconfiguration. This was fixed along with some robustness changes to ensure message forwarding is more consistent e.g. during platform maintenance.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-501-fixes-to-increase-databroker-robustness.md:17]:
```suggestion
Due to an issue, data broker connectors stopped forwarding messages after platform network reconfiguration. This issue has been fixed along with some robustness changes to ensure message forwarding is more consistent, for example, during platform maintenance.  
```

File: content/change-logs/platform-services/cumulocity-10-20-501-fixes-to-increase-databroker-robustness.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-07-23
+title: "Issue with data broker message forwarding after platform network reconfiguration"
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-501-fixes-to-increase-databroker-robustness.md:3]:
```suggestion
title: "Data broker message forwarding no longer stops after platform network reconfiguration"
```

**Review by pink-mongoose — APPROVED** [review]
I like the changes that @BeateRixen has suggested.

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-501-fixes-to-increase-databroker-robustness.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-07-23
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-501-fixes-to-increase-databroker-robustness.md:2]:
```suggestion
date: 
```

**Pull request merged**